engine/table/src/main/java/io/deephaven/engine/table/impl/by/typed/HasherConfig.java

```diff
@@ -103,17 +101,30 @@ public ProbeSpec(String name, String stateValueName, boolean requiresRowKeyChunk
     }
 
     static class BuildSpec {
+        @FunctionalInterface
+        public interface TriConsumer<T, U, V> {
+            void accept(T t, U u, V v);
+        }
```

Comment:
At this point, I would make specific interfaces, in lieu of BiConsumer and TriConsumer.